src/pages/billing/Project.js

```diff
@@ -53,20 +54,29 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
   const [selectedBilling, setSelectedBilling] = useState()
   const [tab, setTab] = useState(query.tab || 'workspaces')
 
+  const [sort, setSort] = useState({ field: 'email', direction: 'asc' })
+
+
```

Comment:
Generally we only have two blank lines after deps, between labelled sections inside a Component, or between components in a long file; could you just glance through your whitespace changes?